Makefile

```diff
@@ -17,8 +17,7 @@ endif
 include $(BOLOS_SDK)/Makefile.defines
 
 APP_LOAD_PARAMS  = --curve secp256k1
-APP_LOAD_PARAMS += --appFlags 0x240
-# APP_LOAD_PARAMS += --appFlags 0x40
+APP_LOAD_PARAMS += --appFlags 0x200
```

Comment:
suggestion(non-blocking): Add a comment explaining the appFlags value used here and possibly the "common load parameters". The other ones are self explanatory already.

Reply:
This is to enable some features on the ledger secure sdk.
This may be a lot to add on the comment, i think the ledger developer portal should be the go to for this kind of information.